Example/Tests/BPKBedgeTest.swift

```diff
@@ -0,0 +1,65 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2022 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import XCTest
+import Backpack
+import FBSnapshotTestCase
+
+class BPKBadgeSnapshotTest: FBSnapshotTestCase {
+    override func setUp() {
+        super.setUp()
+        recordMode = false
+    }
+    
+    private func createStackView() -> UIStackView {
+        let stackView = UIStackView(frame: .zero)
+        stackView.axis = .vertical
+        stackView.alignment = .leading
+        stackView.distribution = .equalCentering
+        stackView.spacing = BPKSpacingMd
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        return stackView
+    }
+    
+    private func createViewWithTypes() -> UIView {
+        let stackView = createStackView()
+        let parentView = UIView(frame: .zero)
+        parentView.backgroundColor = BPKColor.skyGrayTint06
+        parentView.translatesAutoresizingMaskIntoConstraints = false
+        parentView.addSubview(stackView)
+        
+        let badges = [BPKBadgeType.light, .inverse, .outline, .success, .warning, .destructive]
+            .map { BPKBadge(type: $0, message: "Backpack rocks") }
```

Comment:
Do we want to add an `!` to keep in sync with the original test? (Backpack rocks!)

Reply:
this tests was actually like this! it's another test that has the bang!